Update Toolbar component readme

kjellr opened the pull request:
Adding documentation to describe the use and functionality of the Toolbar component.

Thanks @cburton4 and @alexislloyd for drafting this.

kjellr committed:
UpdateToolbar component readme

Adding documentation to describe the use and functionality of the Toolbar component.

Thanks @cburton4 and @alexislloyd for drafting this.

melchoyce reviewed (approved):
Everything looks good to me 👍

Edit: Actually, seeing heading issues in here.

melchoyce commented on packages/components/src/toolbar/README.md:
```diff
@@ -1,5 +1,44 @@
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+
+# Design guidelines
```
Heading level issue

kjellr replied:
Good catch! Just pushed b1e320c to fix that. Might giving it one more review? Thank you!

kjellr committed:
Update heading levels

There should only be one H1: the page title. Adjusting the rest of the levels to match.

chrisvanpatten commented on packages/components/src/toolbar/README.md:
```diff
@@ -1,6 +1,45 @@
+
+### States
+
+**Active and available toolbars**
```
Should these be h4s now similar to #13334?

kjellr replied:
Yes, they should! Updated in b1e320c. Thanks

kjellr committed:
Switch from bold text to headings.

chrisvanpatten reviewed (approved):
Looks great!

melchoyce reviewed (approved):
LGTM!

kjellr commented:
Thank you both!
